<launch>
	
	<group ns="scarborough">
		<node pkg="scarborough" name="imu2" type="IMU2"/>
	</group>
	
	<group ns="scarborough">
		<node pkg="scarborough" name="Arduino" type="ArdI2C"/>
	</group>
	
	<group ns="scarborough">
		<node pkg="scarborough" name="dead_motors" type="Dead_Motors"/>
	</group>
	

</launch>
